third_party/move/move-compiler-v2/src/pipeline/control_flow_graph_simplifier.rs

```diff
@@ -340,6 +357,13 @@ impl ControlFlowGraphCodeGenerator {
         *successors.first().expect("successor block")
     }
 
+    /// Gets the only predecessor of `block`; panics if there is no predecessors.
```

Comment:
is -> are
or
predecessors -> predecessor

Reply:
done